package.json

```diff
@@ -135,9 +135,9 @@
     "build:components:js": "BABEL_ENV=cjs rollup -c ./config/rollup.config.js --format=cjs --environment NODE_ENV:production,FORMAT:cjs",
     "build:components:esm": "BABEL_ENV=esm rollup -c ./config/rollup.config.js --environment NODE_ENV:production,FORMAT:esm",
     "build:components:umd": "rollup -c ./config/rollup.config.js --format=umd --environment NODE_ENV:production,FORMAT:umd",
-    "test": "jest --passWithNoTests --silent",
-    "test:watch": "jest --watch --passWithNoTests",
-    "test:update": "jest --updateSnapshot --passWithNoTests",
+    "test": "jest --passWithNoTests --silent --no-cache",
```

Comment:
I also feel that `TZ=UTC` is not compulsory, but wanted to bring attention. Do you think we can ignore adding that?